lib/srv/db/common/auth.go

```diff
@@ -599,6 +599,10 @@ func shouldUseSystemCertPool(sessionCtx *Session) bool {
 		//
 		// https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-proxy.howitworks.html#rds-proxy-security.tls
 		return true
+
+	case types.DatabaseTypeOpenSearch:
+		// OpenSearch is commonly hosted on AWS and uses Amazon Root CAs.
+		return true
```

Comment:
This comment was added before go1.18 mainly as I was afraid that SystemCertPool will not be able to verify on Windows (though i doubt ppl run database agents on windows).

Personally, now I would prefer SystemCertPool for its simplicity so we don't have to worry about tracking what URL to download (and the URL can change too).

Alternatively, we can append this to the system cert pool (IIRC, we do that for azure)

Reply:
Makes sense. I'm in favor of using the system cert pool as a base but also adding the AWS root CA for all our AWS dbs just in case system cert path is overridden by env vars